Mathlib/Data/Matrix/Notation.lean

```diff
@@ -181,25 +181,26 @@ end DotProduct
 
 section ColRow
 
+variable {ι : Type} [Unique ι]
 @[simp]
-theorem col_empty (v : Fin 0 → α) : col v = vecEmpty :=
+theorem col_empty (v : Fin 0 → α) : col (ι := ι) v = vecEmpty :=
   empty_eq _
 #align matrix.col_empty Matrix.col_empty
 
 @[simp]
-theorem col_cons (x : α) (u : Fin m → α) : col (vecCons x u) = vecCons (fun _ => x) (col u) := by
+theorem col_cons (x : α) (u : Fin m → α) : col (ι := ι) (vecCons x u) = vecCons (fun _ => x) (col u) := by
   ext i j
   refine' Fin.cases _ _ i <;> simp [vecHead, vecTail]
 #align matrix.col_cons Matrix.col_cons
 
 @[simp]
-theorem row_empty : row (vecEmpty : Fin 0 → α) = fun _ => vecEmpty := by
```

Comment:
I think we'd see added `ι` in quite a lot of places. Unless we can tell Lean to use `ι := Fin 1` if it can't infer it otherwise?